apps/style/netsim/style.scss

```diff
@@ -417,6 +417,9 @@ svg {
         text-align: right;
         color: darkgray;
       }
+      .bit_counter.oversized {
+        color: red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.